SocketRocket/SRWebSocket.m

```diff
@@ -1570,7 +1567,7 @@ - (NSOperationQueue *_Nullable)delegateOperationQueue
 
 @end
 
-//#define SR_ENABLE_LOG
+#define SR_ENABLE_LOG
```

Comment:
master branch is ok

(should have checked there first - sorry for the wasted time)